bigquery/google/cloud/bigquery/client.py

```diff
@@ -349,10 +347,10 @@ def create_dataset(self, dataset, exists_ok=False, retry=DEFAULT_RETRY):
         https://cloud.google.com/bigquery/docs/reference/rest/v2/datasets/insert
 
         Args:
-            dataset (Union[ \
```

Comment:
How do the docs render for this change? Run `nox -s docs` and then look at the generated docs in `docs\_build\html`.

I recall previously Sphinx needed the ` \` continuation markers to render this properly.

Reply:
Hm, I didn't attach importance to this, 'cause this slashes are not used all over the place. But looks like you right, deleting them can cause marking problems. Pushed a fix